addons/missionTesting/functions/fnc_displayMenu.sqf

```diff
@@ -0,0 +1,260 @@
+
+        private _createCtrlLine2 = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscLine),-1,CONTROL_GROUP_L];
+        _createCtrlLine2 ctrlSetPosition [0.01,GVAR(yStartCoord),LINE_W,0];
+        _createCtrlLine2 ctrlCommit 0;
+        INCREMENT_YCOORD;
+
+        private _ctrlCreateSectionPassQuestion = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscText),-1,CONTROL_GROUP_L];
+        private _sectionQuestionPassFail = format ["Does this mission pass the %1 requirements?",_sectionTitle];
+        _ctrlCreateSectionPassQuestion ctrlSetText _sectionQuestionPassFail;
+        _ctrlCreateSectionPassQuestion ctrlSetPosition [0.01,GVAR(yStartCoord),0.7,CONTROL_SIZE_H];
+        _ctrlCreateSectionPassQuestion ctrlCommit 0;
+
+        private _idcPassFail = IDC_CBITEMS + _forEachIndex;
+        private _ctrlCreateSectionInstruction = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscCombo),_idcPassFail,CONTROL_GROUP_L];
+        lbCLear _ctrlCreateSectionInstruction;
+        {
+            _ctrlCreateSectionInstruction lbAdd _x;
+        } forEach A_PASSFAIL;
+        _ctrlCreateSectionInstruction lbSetCurSel _sectionPassFail;
+        _ctrlCreateSectionInstruction ctrlSetPosition [0.70,GVAR(yStartCoord),0.12,CONTROL_SIZE_H];
+        _ctrlCreateSectionInstruction ctrlAddEventHandler [QUOTE(LBSelChanged),{_this call FUNC(updateSelectionLB);}];
+        _ctrlCreateSectionInstruction ctrlCommit 0;
+
+        INCREMENT_YCOORD_TEXT;
+        private _createCtrlLine3 = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscLine),-1,CONTROL_GROUP_L];
+        _createCtrlLine3 ctrlSetPosition [0.01,GVAR(yStartCoord),LINE_W,0];
+        _createCtrlLine3 ctrlCommit 0;
+        INCREMENT_YCOORD;
+
+        if(_missionMaker != name player) then {
+            private _ctrlCreateSectionMMNotes = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscText),-1,CONTROL_GROUP_L];
+            _ctrlCreateSectionMMNotes ctrlSetText "NOTES FOR MISSION MAKER:";
+            _ctrlCreateSectionMMNotes ctrlSetPosition [0.01,GVAR(yStartCoord),LINE_W,CONTROL_SIZE_H];
+            _ctrlCreateSectionMMNotes ctrlCommit 0;
+
+            INCREMENT_YCOORD_TEXT;
+
+            private _idcMMNotes = IDC_PASSFAIL + _forEachIndex;
+            private _ctrlCreateSectionMMNotesEdit = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscEditMulti),_idcMMNotes,CONTROL_GROUP_L];
+            _ctrlCreateSectionMMNotesEdit ctrlSetText _sectionNotes;
+            _ctrlCreateSectionMMNotesEdit ctrlSetPosition [0.01,GVAR(yStartCoord),LINE_W,0.15];
+            _ctrlCreateSectionMMNotesEdit ctrlAddEventHandler [QUOTE(KillFocus),{_this call FUNC(updateNotesFlag);}];
+            _ctrlCreateSectionMMNotesEdit ctrlCommit 0;
+
+            INCREMENT_YCOORD + 0.14;
+        };
+};
+
+//Main Function
+
+private _display = createDialog QGVAR(displayMissionTesting);
+
+if (isNull DISPLAY_TESTMENU) exitWith { systemChat "bad display"; };
+
+GVAR(yStartCoord) = 0.01;
+
+private _ctrlBackground = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscBackgroundGUI),-1];
```

Comment:
Any reason to create this in code instead of in config?

Reply:
I haven't much experience with creating it in config. I did this because I know how to do this. But having played with the Eden stuff, I think that I will make converting this a priority in future iterations.